src/server/auth/cmds/cmds.go

```diff
@@ -394,7 +395,11 @@ func WhoamiCmd(ctx context.Context, pachctlCfg *pachctl.Config) *cobra.Command {
 			}
 			fmt.Printf("You are \"%s\"\n", resp.Username)
 			if resp.Expiration != nil {
-				fmt.Printf("session expires: %v\n", resp.Expiration.Format(time.RFC822))
+				if t, err := types.TimestampFromProto(resp.Expiration); err != nil {
+					fmt.Fprintf(os.Stderr, "session exipration time unparseable: %v\n", err)
```

Comment:
Why does one of these go to stdout (implicitly) and one go to stderr?

I think both should be explicit, and probably go to the same place.  If you expect another program to consume it, it should go to stdout.  If it's helpful information, subject to change, it should go to stderr.

Reply:
The parse error goes to stderr, the actual message goes to stdout.  I don't have a problem with using fmt.Printf for things that should go to stdout.  In the follow-up to this, timestamp -> time conversion can't error, so this code path goes away :/